Makefile

```
@@ -22,7 +22,7 @@ build-docker:
 	docker build -f ./Dockerfile -t terraform-validator .
 
 build:
-	GO111MODULE=on go build -o ${build_dir}/${name}
+	GO111MODULE=on CGO_ENABLED=0 go build -ldflags="-w -s" -o ${build_dir}/${name}
```

```suggestion
	GO111MODULE=on CGO_ENABLED=0 go build -o ${build_dir}/${name}
```

Prefer that you set these ldflags when you are making a build for your own production env